monitor/flowgen/flowgen.go

```diff
@@ -45,14 +45,40 @@ func Start(
 		return errors.Wrap(err, "backends")
 	}
 
+	owner := eoa.MustAddress(network.ID, eoa.RoleFlowgen)
+
+	return startWithBackends(ctx, network, backends, owner)
+}
+
+func startWithBackends(
+	ctx context.Context,
+	network netconf.Network,
+	backends ethbackend.Backends,
+	owner common.Address,
+) error {
 	var jobs []types.Job
 
-	result, err := bridgeJobs(network.ID)
+	result, err := bridgeJobs(ctx, network.ID, owner)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "bridge jobs")
 	}
 	jobs = append(jobs, result...)
 
+	if network.ID == netconf.Omega {
+		deposit := big.NewInt(0).Mul(util.MilliEther, big.NewInt(20)) // 0.02 ETH
```

Comment:
I'd follow your existing `bridgeJobs` patter here, so wrap this in a `symbioticJobs` function

Reply:
Will do! Context: I absolutely planned this to do this, but without introducing abstractions for just one job in the same PR.